pkg/controller.v1alpha3/trial/trial_controller_util.go

```diff
@@ -118,6 +130,9 @@ func (r *ReconcileTrial) updateFinalizers(instance *trialsv1alpha3.Trial, finali
 }
 
 func isTrialObservationAvailable(instance *trialsv1alpha3.Trial) bool {
+	if instance == nil {
+		return false
+	}
```

Comment:
just wondering when instance becomes nil ?

Reply:
Never, but prefer to keep to avoid potential crashes.